src/main/java/Sinep/Sinep.java

```diff
@@ -0,0 +1,322 @@
+package Sinep;
+
+import ExceptionHandling.InvalidCommandException;
+import ExceptionHandling.InvalidCommandMessageException;
+import ExceptionHandling.InvalidTaskIndexException;
+
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Objects;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.nio.file.Files;
+import java.io.IOException;
+
+
+public class Sinep {
+    protected static final String FILE_PATH = "src/main/java/Sinep/Sinep.txt";
+    static String line = "_____________________________________________________________________";
+    static String nl = System.lineSeparator();
```

Comment:
Please use a descriptive name for this variable like: `newLine` or `lineSeparator` itself. This enhances the code readability.